source/winVersion.py

```diff
@@ -28,36 +29,40 @@ def isUwpOcrAvailable():
 
 
 WIN10_VERSIONS_TO_BUILDS = {
-	1507: 10240,
-	1511: 10586,
-	1607: 14393,
-	1703: 15063,
-	1709: 16299,
-	1803: 17134,
-	1809: 17763,
-	1903: 18362,
-	1909: 18363,
-	2004: 19041,
-	2009: 19042,
+	"1507": 10240,
+	"1511": 10586,
+	"1607": 14393,
+	"1703": 15063,
+	"1709": 16299,
+	"1803": 17134,
+	"1809": 17763,
+	"1903": 18362,
+	"1909": 18363,
+	"2004": 19041,
+	"20H2": 19042,

```

Comment:
I don't like the sudden API change from build IDs to xxhx strings. Can we somehow support both? (in other words, 1903 is also 19H1)